catalog/tests/test_views.py

```diff
@@ -0,0 +1,281 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from django.contrib.auth.models import User, Permission
+import datetime
+import uuid
+from catalog.models import Author, BookInstance, Book, Genre, Language
+
+class AuthorListViewTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
method này có tác dụng gì?

Reply:
'setUpTestData()' sẽ khởi tạo 1 dữ liệu ban đầu và sẽ dùng cho toàn bộ TestCase